### PR TITLE
Removed unused and duplicate imports

### PR DESCRIPTION
**Pull request by rajpratyush**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!
Closes #6094

### CODE REVIEW

**Comment by abhishalya** [comment]
Please read the checklist and comply to it.

**Comment by Th3nn3ss** [comment]
Do check that you are following the commit guidelines as it's important for a reviewer to understand the improvements being made or bug being fixed.

File: README.rst [hunk 1]
```diff
@@ -35,6 +35,3 @@ coala supports popular programming languages including **Python**, **C/C++**, **
 .. |OpenHub| image:: http://www.openhub.net/p/coala/widgets/project_thin_badge.gif
    :target: https://www.openhub.net/p/coala
 
-.. image:: https://graphs.waffle.io/coala/coala/throughput.svg
- :target: https://waffle.io/coala/coala/metrics/throughput
```
Comment (anshalshukla) [inline README.rst:39]: This is not related to this issue.

**Review by anshalshukla — COMMENTED** [review]
There was already PR related to this issue, you could have cherry-picked that to avoid duplicate work.

**Comment by abhishalya** [comment]
Closing this for irrelevant changes and not following commit guidelines.
